src/main/java/net/twasi/obsremotejava/message/response/inputs/GetInputTracksResponse.java

```diff
@@ -0,0 +1,23 @@
+package net.twasi.obsremotejava.message.response.inputs;
+
+import com.google.gson.JsonObject;
+import lombok.Getter;
+import lombok.ToString;
+import net.twasi.obsremotejava.message.request.Request;
+import net.twasi.obsremotejava.message.response.RequestResponse;
+
+@Getter
+@ToString(callSuper = true)
+public class GetInputTracksResponse extends RequestResponse {
+    private Data responseData;
+
+    public GetInputTracksResponse() {
+        super(Request.Type.GetInputTracks);
+    }
+
+    @Getter
+    @ToString
+    public static class Data {
+        private JsonObject inputAudioTracks;
```

Comment:
its a JsonObject with fields for each audio track. The request sheet says `inputAudioTracks{1, 2, 3, ...}` but since i'm not sure how many there are exactly (and if this can be changed) i indeed put it as a JsonObject.

Reply:
Yeah, I saw the desc in the sheet, but I'm not really sure if it is a json object or array, that's why I would leave a todo. If you're sure it's a json object, disregard my comment

Reply:
Maybe we call it good for now, and verify once it's implemented?

Reply:
Yeah that was indeed what i was thinking